packages/devtools_app/lib/debugger_html_plugin.dart

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// We actually use dart:html here, no shim.
+// This code may only be compiled into the web app.
+import 'dart:ui' as web_ui;
```

Comment:
`dart:ui` should be aliased as `ui` or `dart_ui` not `web_ui`.

Reply:
My goal here is to be very specific: Right now this code will only work in the Flutter web implementation of dart:ui.

Happy to consider alternatives, but it's important to get that across to readers of this code.

Reply:
how about
`dart_ui_web`
`web_ui` made me think `dart:html` but it might be just me.